java/json/src/main/java/com/microsoft/azure/schemaregistry/kafka/json/JsonSerializerException.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.schemaregistry.kafka.json;
+
+import org.apache.kafka.common.errors.SerializationException;
+
+/**
+ * Custom error class for exceptions thrown in Json serialization/deserialization steps
+ */
+public class JsonSerializerException extends SerializationException {
```

Comment:
we can call this JsonSerializationException, so that we can use it in both Serializer and Deserializer cases.